### DISCUSSION TITLE
URL for Group View

### DISCUSSION THREAD

**Discussion by psCLE** [blakeblackshear/frigate#14550] [Ideas]
Hello. I am working on a cost-effective way to view cameras on a TV driven by Frigate and use [OliveTin](https://olivetin.app) controlled by a mobile phone to make changes to the view. While viewing Birdseye, viewing all the cameras, and viewing an individual camera is a snap, I'd also like the user to have an option to view camera groups. For example, a group of cameras that look over the pool and patio. As it is now, when I choose a group in the Frigate UI, the group displays in the window as expected but the URL doesn't change. This makes it impossible for my scheme to load a Frigate camera group.

My feature request would be a URL for camera group views. While full screen Birdseye is /#birdseye and a full screen single camera view is /#Camera_01, a camera group view URL could be /pool_group.

**Comment by hawkeye217** [comment]
This is already available. Use the url:

`http://frigate_ip:5000/?group=<group_name>`

Note that the group name is case sensitive.

**Reply by psCLE** [reply]
Ok, perfect. Thanks for pointing this out to me.

**Reply by hawkeye217** [reply]
Happy to help!
